stampede.js

```diff
@@ -105,8 +105,8 @@ class Stampede {
 
     await (options.upsert ? this._adapter.update(key,payload,expiry) : this._adapter.insert(key,payload,expiry));
 
-    const finalize = async(err, d) => {
-      if (err) {
+    const finalize = async (err, d) => {
+      if (err !== null && err !== undefined) {
```

Comment:
If the `err` was an empty string, a null value was cached.  This happened to us when using `got` and `http2`, which does not define a statusMessage and was therefore passing in an empty `err` string.